bcda/auth/client/ssas_test.go

```diff
@@ -14,12 +22,41 @@ func (s *SSASClientTestSuite) TestNewSSASClient() {}
 
 func (s *SSASClientTestSuite) TestCreateSystem() {}
 
-func (s *SSASClientTestSuite) TestGetPublicKey() {}
+func (s *SSASClientTestSuite) TestGetPublicKey() {
```

Comment:
this is a good method, but do we have test cases for the environment vars not being set? Like a check for the error strings that'd we expect in our logging.